src/manager/app/models/volume.py

```diff
@@ -0,0 +1,41 @@
+import os
+from typing import Optional
+
+from pydantic import BaseModel, computed_field
+
+STATE_HOST_DIR = os.getenv("STATE_HOST_DIR")
+
+
+class RAGAppVolumeConfig(BaseModel):
+    name: str
+    container_data_mount_path: str = "/app/data"
+    container_config_mount_path: str = "/app/config"
+    host_mount_prefix: Optional[str] = STATE_HOST_DIR
+
+    @computed_field
+    @property
+    def host_mount_path(self) -> str:
+        return f"{self.host_mount_prefix}/{self.name}"
```

Comment:
```suggestion
        return f"{self.host_mount_prefix}/ragapps/{self.name}"
```

Reply:
then `ragapps` can be removed below

Reply:
but this should then be `app_mount_path` and not `host_mount_path`

Reply:
Great catch! I updated this! But it's `host_mount_path` as it's the the path from host.